### ISSUE TITLE
New Drive Documents Open in New Window

### ISSUE THREAD

**Issue by chriskol** [chriskol/Google-Drive-Desktop-OSX#1]
New Drive documents should probably open in the same window.  Probably a way to force targets to open in same window.  Will look into later as this was a quick creation but open to someone fixing it

**Comment by ranjanan** [comment]
Yeah sure. I'm a complete web technologies newbie but would you know the right place to look up a fix? Maybe some macgap documentation?

**Comment by chriskol** [comment]
I don't think it's entirely macgap documentation necessarily and tbh I'm having difficulty finding the macgap documentation.  Maybe the project was abandoned?  It's based off javascript and html for the most part though so I think.  Looks like the docs web page is filled with spam but I think [this should work](https://github.com/MacGapProject/documentation)?  Also [here](https://github.com/MacGapProject) in general.
